lighthouse-core/audits/installable-manifest.js

```diff
@@ -5,38 +5,112 @@
+  /** Error message explaining that the manifest display property must be one of 'standalone', 'fullscreen', or 'minimal-ui'. */
+  'manifest-display-not-supported': `Manifest 'display' property must be one of 'standalone', 'fullscreen', or 'minimal-ui'`,
+  /** Error message explaining that the manifest could not be fetched, might be empty, or could not be parsed. */
+  'manifest-empty': `Manifest could not be fetched, is empty, or could not be parsed`,
+  /** Error message explaining that no matching service worker was detected,
+   * and provides a suggestion to reload the page or check whether the scope of the service worker
+   * for the current page encloses the scope and start URL from the manifest. */
+  'no-matching-service-worker': `No matching service worker detected. You may need to reload the page, or check that the scope of the service worker for the current page encloses the scope and start URL from the manifest.`,
+  /**
+  * @description Error message explaining that the manifest does not contain a suitable icon.
+  * @example {192} value0
+  */
+  'manifest-missing-suitable-icon': `Manifest does not contain a suitable icon - PNG, SVG or WebP format of at least {value0}\xa0px is required, the sizes attribute must be set, and the purpose attribute, if set, must include "any" or "maskable".`,
+
+  /**
+  * @description Error message explaining that the manifest does not supply an icon of the correct format.
+  * @example {192} value0
+  */
+  'no-acceptable-icon': `No supplied icon is at least {value0}\xa0px square in PNG, SVG or WebP format`,
+
+  /** Error message explaining that the downloaded icon was empty or corrupt. */
+  'cannot-download-icon': `Downloaded icon was empty or corrupted`,
+  /** Error message explaining that the downloaded icon was empty or corrupt. */
+  'no-icon-available': `Downloaded icon was empty or corrupted`,
+  /** Error message explaining that the specified application platform is not supported on Android. */
+  'platform-not-supported-on-android': `The specified application platform is not supported on Android`,
+  /** Error message explaining that a Play store ID was not provided. */
+  'no-id-specified': `No Play store ID provided`,
+  /** Error message explaining that the Play Store app URL and Play Store ID do not match. */
+  'ids-do-not-match': `The Play Store app URL and Play Store ID do not match`,
+  /** Error message explaining that the app is already installed. */
+  'already-installed': `The app is already installed`,
+  /** Error message explaining that a URL in the manifest contains a username, password, or port. */
+  'url-not-supported-for-webapk': `A URL in the manifest contains a username, password, or port`,
+  /** Error message explaining that the page is loaded in an incognito window. */
+  'in-incognito': `Page is loaded in an incognito window`,
+  // TODO: perhaps edit this message to make it more actionable for LH users
```

Comment:
This will get lost in here, so should this be filed as an issue or is it covered enough now that it could be removed?

Reply:
done! #11830 Yeah these are messages that we might want to communicate with the installability/devtools folks about, to get a clearer definition of for LH